course-v2/assets/css/course-image-section.scss

```diff
@@ -5,7 +5,6 @@
 
   border: 1px solid $medium-gray;
   width: 100%;
-  object-fit: cover;
```

Comment:
Removing this makes the course image stretch disproportionately in its container:

![image](https://user-images.githubusercontent.com/12089658/192340995-b8386854-99e9-48a1-968c-8beefca5e4da.png)

Might we want to use `object-fit: contain;` instead?

Reply:
We might also consider removing the max height on the image of 200px.  This is with `object-fit: contain` and the max height removed:

![image](https://user-images.githubusercontent.com/12089658/192342528-0a42300c-690a-40cc-8d25-3e100321d9ed.png)